libs/blockchain-watcher/feature-app/src/lib/Enum/EnumContractEventType.ts

```diff
@@ -0,0 +1,5 @@
+export enum EnumContractEventType {
```

Comment:
The enum names seem a little verbose, is `ContractEventType` or `ContractEvent` ok? The reasoning is `Enum` is already implied by the type, and `Type` is a little confusing, since it's not a typescript type.

Reply:
The `Enum` prefix is from C# coding style, will change it to `ContractEventType` if you think it's better. But I'll said the prefix is useful when we have a good auto-complete engine.

Reply:
We do have a good autocomplete engine, but unfortunately it's built into VSCode 😅

You can keep the enum prefix, I don't feel that strongly about it.